src/decode.rs

```diff
@@ -2272,6 +2247,8 @@ fn decode_b(
         }
     } else {
         // inter-specific mode/mv coding
+        let mv_prec = || -> i32 { frame_hdr.hp as i32 - frame_hdr.force_integer_mv as i32 };
```

Comment:
Nit: I think this should work, too.

```suggestion
        let mv_prec = || frame_hdr.hp as i32 - frame_hdr.force_integer_mv as i32;
```